tests/filter/empty_reimport.t

```diff
@@ -48,11 +48,11 @@
   $ git checkout other_branch
   Switched to branch 'other_branch'
 
-  $ echo content > pre/blafile1
+  $ echo content > pre/blah-file1
   $ git add pre
   $ git commit -m "more change on other" 1> /dev/null
 
-  $ echo content > pre/blefile2
+  $ echo content > pre/blah-file2
```

Comment:
I can see that the existing code used `bla` and `ble`, but there didn't seem to be any particular need for the two different spellings given the file suffixes `1` and `2`.